Rakefile

```diff
@@ -12,4 +12,35 @@ require "bundler/gem_tasks"
 RuboCop::RakeTask.new
 RSpec::Core::RakeTask.new
 
-task default: %i[rubocop spec]
+require "govuk_web_banners/validators/recruitment_banner"
+require "rainbow"
+
+desc "show errors in the live config"
+task :check_config do
+  validator = GovukWebBanners::Validators::RecruitmentBanner.new(GovukWebBanners::RecruitmentBanner.all_banners)
+
+  if !validator.valid?
+    puts Rainbow("\nLive config contains errors!").red
+    validator.errors.each_key do |key|
+      puts(key)
+      validator.errors[key].each { |error| puts(" - #{error}") }
+    end
+    puts
+    exit(1)
+  elsif validator.warnings?
```

Comment:
It was intentional. We could do errors and warnings in one blob, but I thought it would be better to keep it to one of three states at a time (errors you have do do something about now, warnings you might be able to ignore, or all clear).